Reconstruction/RecTracker/src/components/FullInnerLayerGraphTool.h

```diff
@@ -0,0 +1,23 @@
+#ifndef RECTRACKER_FULLINNERLAYERGRAPHTOOL_H
+#define RECTRACKER_FULLINNERLAYERGRAPHTOOL_H
+
+// from Gaudi
+#include "GaudiAlg/GaudiTool.h"
+
+// FCCSW
+#include "FWCore/DataHandle.h"
+#include "RecInterface/ILayerGraphTool.h"
+
+#include "tricktrack/CMGraph.h"
+
+
```

Comment:
This is for use with FCChhTracker, right?
I'd suggest then to disentangle it from RecTracker and create RecFCChhTracker.
And I'd add `Tracker` to the name, maybe instead of `Full` or `Layer` (or both).